CMakeLists.txt

```diff
@@ -8,7 +8,8 @@
 # SPDX-License-Identifier: BSD-3-Clause
 #
 
-cmake_minimum_required(VERSION 3.22)
+# TODO: Bump to 3.25 or keep 3.22?
```

Comment:
What is the benefit of bumping to 3.25?

Reply:
It just gives access to a newer form of `try_compile` which doesn't require specifying the build directory: https://cmake.org/cmake/help/v3.25/command/try_compile.html. Not a must at all, I just went with this for the first iteration until we decide what to do exactly.

Reply:
Needs an update to the Spack package too if we go forward with this update.

Reply:
I've changed it to use the older form of `try_compile` and reverted the version bump.